services/payments/src/simcore_service_payments/models/db.py

```diff
@@ -23,6 +24,7 @@
     "wallet_id": 123,
     "comment": "This is a test comment.",
     "invoice_url": None,
+    "stripe_invoice_id": None,
```

Comment:
I see you have a model of how it looks like after init, is there such a thing as well for once you get all the stripe stuff back?

Reply:
There will be one in the next PR, as Stripe client will be introduced